jest.config.js

```diff
@@ -20,6 +20,9 @@ module.exports = {
 	],
 	coverageDirectory: 'coverage',
 	coverageReporters: ['text', 'cobertura'],
+	moduleNameMapper: {
+		'react-pdf/dist/esm/entry.webpack': 'react-pdf'
```

Comment:
I think that removing this import from common will also make this mapping unnecessary

Reply:
See https://github.com/Zextras/carbonio-files-ui-common/pull/13#discussion_r845307653